### ISSUE TITLE
RAID Resets

### ISSUE THREAD

**Issue by eabyss-source** [WoWManiaUK/Blackwing-Lair#4244]
**What is happening:**
Currently raids are automatically resetting on Sunday Evening. However they are also once again reset by the GM Team when they do the weekly Valor Point Reset on Tuesday

**What should happen:**
Reset for VP and Instances should be done at the same time and only once a week.

**Comment by Echinaceax** [comment]
This has be told to Janitor from internal report, but thanks @eabyss-source

**Comment by Echinaceax** [comment]
**Update on this issue** 
spoken about here before https://github.com/WoWManiaUK/Blackwing-Lair/issues/3064

Since GMs stopped resetting this manually on **wednesday 00:00** this week for retest for me there is a new issue 
- the Sunday reset has stopped working to now 

Did not reset at 6am as told in comment by janitor https://github.com/WoWManiaUK/Blackwing-Lair/issues/3064#issuecomment-535478741

SS taken today ( server didnt not reset itself) 
ive informed the GMs to manually reset as they did before for now, 
something is messing up the resets somewhere

i logged off Sunday at 12:36am and checked this again today 10 mins ago 
![no sunday reset](https://user-images.githubusercontent.com/39439201/85946802-40406500-b947-11ea-99f7-582f5b96881b.jpg)

note: as you can see the weekly RDF is reset as its suppost too but the Weekly valor cap didnt

**Comment by Echinaceax** [comment]
### Update
If players of US times zone can Check the Sunday/Wednesday (ST) reset this week and next week please 

 what we want to work out is: if the GMs dont manually reset wednesday 00:00 next week, if the 6am ST does it its self on wednesday and stop sundays reset,

its been reset for this week manually by the GM again, but next week i will be informing the GM not to manually reset next week (wednesday 00:00) so we can get this fixed properly

**Comment by Echinaceax** [comment]
confirmed by Hammer
NO reset automaticily done by server today 
up to 06:37 

this needs fixing @wow-mania
